package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/update.js

```diff
@@ -10,8 +10,8 @@ upS.SelF="Selecione o firmware";
 upS.Prsv="Manter Configurações (<b>não recomendado</b>)";
 
 //javascript
-upS.SelErr="ERRO: Você deve selecionar um arquivo de firmware.";
-upS.Warn2="A atualização de firmware apagará completamente suas configurações, e pode, eventualmente, inutilizar (&quot;brickar&quot;) seu roteador se o arquivo de firmware selecionado for inválido. Deseja continuar ?";
+// upS.SelErr="ERRO: Você deve selecionar um arquivo de firmware.";
```

Comment:
Are you sure you want to comment this one out? It is used in the update page

Reply:
Not at all. I was testing something real quick and apparently forgot to uncomment afterward. 

Fixed, thanks for pointing it out.